extensions/binding/org.eclipse.smarthome.binding.hue/src/main/java/org/eclipse/smarthome/binding/hue/handler/HueBridgeHandler.java

```diff
@@ -58,7 +65,7 @@
  * @author Stefan Bußweiler - Added new thing status handling
  * @author Jochen Hiller - fixed status updates, use reachable=true/false for state compare
  */
-public class HueBridgeHandler extends BaseBridgeHandler {
+public class HueBridgeHandler extends BaseBridgeHandler implements ConfigStatusProvider {
```

Comment:
Wasn't the ConfigStatusThingHandler explicitly introduced to avoid handlers to implement all methods by themselves? I see that we have a bridge handler here and that it thus cannot extend two different parent classes, but this only shows that we missed to also introduce a ConfigStatusBridgeHandler, right? That should be fairly easy to do.